bc_obps/registration/tests/test_utils.py

```diff
@@ -1,16 +1,15 @@
+from service.application_access_service import ApplicationAccessService
```

Comment:
Looks like we can split a bunch of these tests into the individual test file for the services

Reply:
Also have a separate tech debt ticket for this: https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=54068291 (there's a list of tech debt tickets in the PR description; this PR is already pretty big)